static/app/components/dropdownAutoComplete/list.tsx

```diff
@@ -105,10 +109,8 @@ const List = ({
           // because the items shift around the list
           key={`${item.value}-${index}`}
           item={item}
-          itemSize={itemSize}
-          highlightedIndex={highlightedIndex}
-          inputValue={inputValue}
-          getItemProps={getItemProps}
+          isActive={item.index === highlightedIndex}
```

Comment:
`isActive` instead of passing in the `highlightedIndex` is what helps us not need to re-render each time, since `isActive` will just change from `true` to `false`, unlike `highlightedIndex` which will change every time